crd-bootstrap/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: crd-bootstrap
+description: This is the official Helm chart for the Project crd-bootstrap.
+type: application
+version: 0.1.0 # will be replaced by the publishing workflow.
```

Comment:
I d suggest to set them to 0.0.0 to clearly mark that those versions are just placeholders.